doc/pages/architecture/090-database.md

```diff
@@ -150,52 +159,96 @@ CREATE TABLE <realm name>.interfaces (
   aggregation int,
   automaton_transitions blob,
   automaton_accepting_states blob,
-  description varchar,
-  doc varchar,
+  description text,
+  doc text,
 
   PRIMARY KEY (name, major_version)
 );
 ```
 
+```sql
+CREATE TABLE <realm name>.simple_triggers (
+  object_id uuid,
+  object_type int,
+  parent_trigger_id uuid,
+  simple_trigger_id uuid,
+  trigger_data blob,
+  trigger_target blob,
+
+  PRIMARY KEY ((object_id, object_type), parent_trigger_id, simple_trigger_id)
+);
+```
+
+```sql
+
+CREATE TABLE <realm name>.individual_datastreams (
+    device_id uuid,
+    interface_id uuid,
+    endpoint_id uuid,
+    path text,
+    value_timestamp timestamp,
+    reception_timestamp timestamp,
+    reception_timestamp_submillis smallint,
+    binaryblob_value blob,
+    binaryblobarray_value list<blob>,
+    boolean_value boolean,
+    booleanarray_value list<boolean>,
+    datetime_value timestamp,
+    datetimearray_value list<timestamp>,
+    double_value double,
+    doublearray_value list<double>,
+    integer_value int,
+    integerarray_value list<int>,
+    longinteger_value bigint,
+    longintegerarray_value list<bigint>,
+    string_value text,
+    stringarray_value list<text>,
+    PRIMARY KEY ((device_id, interface_id, endpoint_id, path), value_timestamp, reception_timestamp, reception_timestamp_submillis)
+) 
+```
+
 ```sql
 CREATE TABLE <realm name>.individual_properties (
   device_id uuid,
   interface_id uuid,
   endpoint_id uuid,
-  path varchar,
+  path text,
   reception_timestamp timestamp,
   reception_timestamp_submillis smallint,
-
   double_value double,
   integer_value int,
   boolean_value boolean,
   longinteger_value bigint,
-  string_value varchar,
+  string_value text,
   binaryblob_value blob,
   datetime_value timestamp,
   doublearray_value list<double>,
   integerarray_value list<int>,
   booleanarray_value list<boolean>,
   longintegerarray_value list<bigint>,
-  stringarray_value list<varchar>,
+  stringarray_value list<text>,
   binaryblobarray_value list<blob>,
   datetimearray_value list<timestamp>,
 
   PRIMARY KEY((device_id, interface_id), endpoint_id, path)
 );
+
 ```
 
+For object datastream interfaces, generated upon interface creation
+
 ```sql
-CREATE TABLE <realm name>.simple_triggers (
-  object_id uuid,
-  object_type int,
-  parent_trigger_id uuid,
-  simple_trigger_id uuid,
-  trigger_data blob,
-  trigger_target blob,
+CREATE TABLE <realm name>.<interface_name>_<major_version> (
```

Comment:
The table name is not `<interface_name>_<interface_major>` (see https://github.com/astarte-platform/astarte_core/blob/685ca10c7a07cc9806f2c6fc7ec2ed1b4d23cbec/lib/astarte_core/cql_utils.ex#L32-L56), for example Scylla/Cassandra tables do not allow dots(`.`) in their name, while interfaces do. Change this and maybe and maybe add some information on table name generation